home/templates/home/home_page.html

```diff
@@ -1,11 +1,41 @@
 {% extends "base.html" %}
 {% load static %}
 {% load wagtailcore_tags %}
-
+ 
```

Comment:
Oh! A blank space on a line for itself will create errors in the "linting" process, you can set your editor to avoid this or use `pre-commit` (see the README file)